web/src/message_edit.js

```diff
@@ -867,11 +868,30 @@ export function save_inline_topic_edit($row) {
         return;
     }
 
+    const $message_header = $row.find(".message_header").expectOne();
+    const stream_id = Number.parseInt($message_header.attr("data-stream-id"), 10);
+    const stream_topics = stream_topic_history.get_recent_topic_names(stream_id);
```

Comment:
So this API does not check with the server whether we have complete server data on topics in the stream. 

Are we expecting that data to usually be available because the typeahead calls `stream_topic_history_util.get_server_history`? I know the compose topic typeahead calls that, but was not sure if the topic-edit one does; if it doesn't, we should add a commit to do that.